### ISSUE TITLE
sqlite3 gem error on freshly cloned lab

### ISSUE THREAD

**Issue by ivalentine** [learn-co-curriculum/rails-hello-world-lab#18]
Running `learn` on a freshly cloned copy of this lab throws this error:

```
An error occurred while loading ./spec/features/hello_world_spec.rb.
Failure/Error: ActiveRecord::Migration.maintain_test_schema!

Gem::LoadError:
  Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/connection_adapters/connection_specification.rb:176:in `rescue in spec'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/connection_adapters/connection_specification.rb:173:in `spec'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/connection_handling.rb:53:in `establish_connection'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/railtie.rb:125:in `block (2 levels) in <class:Railtie>'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:69:in `instance_eval'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:60:in `with_execution_control'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:65:in `execute_hook'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:50:in `block in run_load_hooks'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:49:in `each'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/lazy_load_hooks.rb:49:in `run_load_hooks'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/base.rb:324:in `<module:ActiveRecord>'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/base.rb:24:in `<top (required)>'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/dependencies.rb:293:in `require'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/dependencies.rb:293:in `block in require'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/dependencies.rb:259:in `load_dependency'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activesupport-5.0.7.1/lib/active_support/dependencies.rb:293:in `require'
# /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/migration.rb:591:in `maintain_test_schema!'
# ./spec/rails_helper.rb:28:in `<top (required)>'
# ./spec/features/hello_world_spec.rb:1:in `require'
# ./spec/features/hello_world_spec.rb:1:in `<top (required)>'
# ------------------
# --- Caused by: ---
# Gem::LoadError:
#   can't activate sqlite3 (~> 1.3.6), already activated sqlite3-1.4.0. Make sure all dependencies are added to Gemfile.
#   /Users/ian/.rvm/gems/ruby-2.3.4/gems/activerecord-5.0.7.1/lib/active_record/connection_adapters/sqlite3_adapter.rb:7:in `<top (required)>'
No examples found.

Finished in 0.03354 seconds (files took 6.18 seconds to load)
0 examples, 0 failures, 1 error occurred outside of examples
```

Reported by @rachelwyatt.

**Comment by HumzahChoudry** [comment]
The fix for this is to add `gem 'sqlite3', '1.3.6'`

**Comment by maxwellbenton** [comment]
HI @ivalentine, thank you for bringing this issue to our attention. This issue has been fixed in all labs using sqlite3.
